Parking-Zone/Services/ReservationService.cs

```diff
@@ -20,5 +20,11 @@ public IEnumerable<Reservation> GetByAppUserId(string appUserId)
             return _repository.GetAll()
                 .Where(x => x.AppUserId == appUserId);
         }
+
+        public void Prolong(Reservation reservation, int newDuration)
```

Comment:
```suggestion
        public void Prolong(Reservation reservation, int extraHours)
```

This is not a new duration. You are not replacing old duration with new one. You are adding extra (additional) hours to existing one. So name the variable properly.